test/Ocelot.UnitTests/Configuration/UpstreamTemplatePatternCreatorTests.cs

```diff
@@ -42,7 +42,7 @@ public void should_use_re_route_priority()
 
             this.Given(x => x.GivenTheFollowingFileRoute(fileRoute))
                 .When(x => x.WhenICreateTheTemplatePattern())
-                .Then(x => x.ThenTheFollowingIsReturned("^(?i)/orders/.+$"))
+                .Then(x => x.ThenTheFollowingIsReturned($"^(?i)/orders(?:|/{UpstreamTemplatePatternCreator.RegExMatchZeroOrMoreOfEverything})$"))

```

Comment:
Quite long sentence which hard to read! 
```suggestion
                private const string MatchEverything = UpstreamTemplatePatternCreator.RegExMatchZeroOrMoreOfEverything;
                .Then(x => x.ThenTheFollowingIsReturned($"^(?i)/orders(?:|/{MatchEverything})$"))
```